src/content/plugins/internal-plugins.md

```diff
@@ -168,10 +151,6 @@ Make webpack_public_path, webpack_require, webpack_modules and webpack_chunk_loa
 
 Currently useless. Ensures compatibility with other module loaders.
 
-### ConsolePlugin
```

Comment:
It's moved into `NodeSourcePlugin` in https://github.com/webpack/webpack/commit/c4140aac51ace25944a74d0cbad00ca7cf5540f4